src/LocalRegistry.jl

```diff
@@ -607,4 +607,19 @@ function gitlab(branch, pkg, new_package, repo, commit)
     return branch, push_options
 end
 
+function gitcmd(path::AbstractString, gitconfig::Dict)
+    if Sys.iswindows()
+        # Workaround for https://github.com/GunnarFarneback/LocalRegistry.jl/issues/76.
+        # Cf. https://github.com/ziglang/zig/issues/16526#issuecomment-1648706259
+        cmd = ["cmd", "/c", "git", "-C", path]
+    else
+        cmd = ["git", "-C", path]
+    end
+    for (n,v) in gitconfig
+        push!(cmd, "-c")
+        push!(cmd, "$n=$v")
+    end
+    Cmd(cmd)
+end
+
```

Comment:
This does indeed seem to solve the problem. A few things do some to mind:

- Wouldn't it be neater to explicitly overwrite the gitcmd function from RegistryTools, that way also acknowleding their work in writing the function?
- It might be better to raise the issue with them instead of fixing it here.
- Should there be some form of backward compatibility made available? I'm not sure if `cmd /c git -C path` will always act identically to `git -C path` on Windows, besides the REPL things...

Reply:
I have confidence in my own test suite but all things Windows are a mystery to me so I don't want to push it on anyone else. The biggest reason to duplicate `gitcmd` though is that it will be needed when I get back to #38 anyway.

Overwriting an imported function is not a good practice, both for precompilation and piracy reasons, so that's not an option.